imports/ui/helpers.coffee

```diff
@@ -37,4 +42,10 @@ UI.registerHelper 'incidentToText', (incident)->
       @locations.map(formatLocation).slice(0, -1).join(", ") +
       ", and " + formatLocation(@locations.slice(-1)[0])
     )
-  return "#{incidentDescription} in #{formattedLocations} #{formatTimeRange(@timeRange)}"
+  result = "#{incidentDescription} in #{formattedLocations}"
+  if @timeRange
```

Comment:
With the addition of date ranges in a recent PR, this variable should be "dateRange" and the function should now be "formatDateRange"